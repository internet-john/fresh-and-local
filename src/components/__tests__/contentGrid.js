import React from 'react';
import renderer from 'react-test-renderer';

import ContentGrid from '../content/contentGrid';

describe('ContentGrid', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ContentGrid />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
