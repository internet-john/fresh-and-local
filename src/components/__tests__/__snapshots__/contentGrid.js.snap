// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentGrid renders correctly 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-spacing-xs-1"
>
  <div
    className="MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
    />
  </div>
</div>
`;
